tags/faq/lwjgl.ytag

```diff
@@ -4,4 +4,5 @@ type: text
 
 If you're getting `java.lang.UnsatisfiedLinkError: Failed to locate library: lwjgl.dll` when running your project:
   1. delete `<project dir>/.gradle/loom-cache/launch.cfg`
```

Comment:
```suggestion
  1. Delete `<project dir>/.gradle/loom-cache/launch.cfg`
```

Reply:
help how do I close this lol

Reply:
I already closed it by resolving it, don't worry about it :P

Reply:
ah, apparently I had to refresh the page to see it being resolved